pkg/registry/apis/iam/register.go

```diff
@@ -65,6 +65,7 @@ func RegisterAPIService(
 		display:            user.NewLegacyDisplayREST(store),
 		reg:                reg,
 		enableAuthZApis:    features.IsEnabledGlobally(featuremgmt.FlagKubernetesAuthzApis),
+		enableDualWriter:   true,
```

Comment:
Just double-checking, should this be coming from the config / feature toggles?

Reply:
I guess the dual writer mode already comes from config and given the MT instantiation uses `NewAPIService` I think it's ok to have it set to `true` here.

Reply:
That our (thanks @charandas) thinking as well.